src/script/util/KeyboardUtil.ts

```diff
@@ -136,6 +136,13 @@ export const offEscKey = (handler: KeyboardHandler) => {
   }
 };
 
+export const handleKeyDown = (event: React.KeyboardEvent<HTMLDivElement>, callback: () => void) => {
+  if (event.key === KEY.ENTER || event.key === KEY.SPACE) {
```

Comment:
maybe should also use the isSpaceKey(event) || isEnterKey(event) syntax of the methods? but im okay with this also